src/python/pants/java/junit/junit_xml_parser.py

```diff
@@ -100,20 +100,20 @@ def combined_indexer(tgt):
 class ParseError(Exception):
   """Indicates an error parsing a junit xml report file."""
 
-  def __init__(self, junit_xml_path, cause):
+  def __init__(self, xml_path, cause):
     super(ParseError, self).__init__('Error parsing test result file {}: {}'
-                                     .format(junit_xml_path, cause))
-    self._junit_xml_path = junit_xml_path
+                                     .format(xml_path, cause))
+    self._xml_path = xml_path
```

Comment:
This change was needed so this ParseError has the same properties as the ParseError definied in `src/python/pants/task/testrunner_task_mixin.py`.  I'm not sure if we would rather consolidate these into one ParseError class or not?   Either of these errors can be thrown when parsing an xml file and the `src/python/pants/backend/jvm/tasks/junit_run.py` file handles them with the following method, which expects xml_path. 

```
    def parse_error_handler(parse_error):
      # Just log and move on since the result is only used to characterize failures, and raising
      # an error here would just distract from the underlying test failures.
      self.context.log.error('Error parsing test result file {path}: {cause}'
                             .format(path=parse_error.xml_path, cause=parse_error.cause))
```

Reply:
Oh, so this fixes a bug with junit_run where it didn't work for both exceptions? Seems reasonable. If they're both used to parse junit xml files, I think consolidating them makes sense, but maybe not part of this version bump.

Also, is this change related to the version bump, or could it be broken out along with the consolidation?

Reply:
Yes.  Since either `ParseError` could be thrown, this fixes the logger when the `junit_xml_parser`'s `ParseError` is thrown.

It's not strictly related to the version bump, but it was exposed if you don't include the extra shading exclusions for the junit-runner.

Reply:
Oh. I see. That's fine then.